webapp/components/configuration/listMenuConfiguration.js

```diff
@@ -0,0 +1,137 @@
+import React from "react";
+import {
+  bgRegular,
+  bgDark,
+  font,
+  primaryColor,
+  gutter,
+  radius,
+} from "../../theme";
+
+const style = {
+  standard: {
+    border: 0,
+    background: "transparent",
+    padding: "0",
+    width: "20px",
+    height: "20px",
+    boxShadow: "none",
+    minWidth: "10px",
+    fontSize: "10px",
+    hr: {},
+  },
+  lighter: { background: primaryColor },
+  padding: {
+    fontSize: 15,
+    paddingTop: `calc(${gutter} / 2)`,
+    paddingBottom: `calc(${gutter} / 2)`,
+  },
+};
+
+const topLevelMenuItemStyle = {
+  standard: { background: "transparent" },
+  hover: {},
+};
+
+const firstItemCustom = { fontWeight: "bold" };
+
+const firstItemStyle = {
+  standard: { ...topLevelMenuItemStyle.standard, ...firstItemCustom },
+  hover: { ...topLevelMenuItemStyle.hover, ...firstItemCustom },
+};
+
+const hiddenMenuItemStyle = {
+  standard: { display: "none" },
+  hover: {},
+};
+
+export const listMenuConfigurations = (Instances, entity) => {
+  const color = Instances.getInstance(entity.path).color || "#ffffff";
+  const isImage = entity.type === "ImageSeries";
+  const config = {
+    global: {
+      color: "white",
+      subMenuOpenOnHover: true,
+      menuOpenOnClick: true,
+      menuPadding: 0,
+      fontFamily: font,
+      menuFontSize: "14",
+      subMenuFontSize: "12",
+      background: bgRegular,
+      buttonsStyle: {
+        standard: style.standard,
+        position: "relative",
+        hover: {
+          ...style.standard,
+          ...style.lighter,
+        },
+      },
+      labelsStyle: {
+        standard: { ...style.padding },
+        hover: {
+          ...style.lighter,
+          ...style.padding,
+        },
+      },
+      drawersStyle: {
+        standard: {
+          top: 10,
+          backgroundColor: bgDark,
+          borderRadius: 0,
+          color: "#ffffff",
+          fontSize: 14,
```

Comment:
It would be nice to avoid these kind of custom css and have the menu component using the main theme by default. Surely an improvement to keep in mind on the Geppetto client side.